vector/src/main/java/im/vector/app/features/spaces/SpaceSummaryController.kt

```diff
@@ -181,7 +181,8 @@ class SpaceSummaryController @Inject constructor(
         }
     }
 
-    private fun buildSubSpace(summaries: List<RoomSummary>?,
+    private fun buildSubSpace(idPrefix: String,
```

Comment:
small nit pick - it could help make this a little easier to understand if we highlight the parent/child space relationship  
what do you think?

```kotlin
fun buildSubSpace(parentId: String) {
  val id = "$parent:$child"

  spaceSummaryItem {
    id(id)
  }

  subSpaces.forEach { child ->
    buildSubSpace(parent = id)
  }
}
```